src/utilities.ts

```diff
@@ -190,24 +190,19 @@ function getWordAtPosition(document, position) {
     return text;
 }
 
-async function loadFileIfNamespaceNotExist(doc) {
+async function CreateNamespaceFromDocumentIfNotExists(doc) {
```

Comment:
We name functions using `camelCase`, in our undocumented code convention. 😄

Reply:
Good point! Changed...